pkg/e2e/openshift/osdclusterready.go

```diff
@@ -0,0 +1,57 @@
+// Package openshift contains the OpenShift extended test suite.
+package openshift
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/osde2e-common/pkg/clients/openshift"
+	"github.com/openshift/osde2e/pkg/common/label"
+	batchv1 "k8s.io/api/batch/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var suiteName = "OSD Cluster Ready"
+var timeout = "30m"
+var namespace = "openshift-monitoring"
+var jobname = "osd-cluster-ready"
```

Comment:
please make these constants and I feel like it should be formatted differently via gofmt